src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -314,6 +380,16 @@ slope_thresh = 1/15
 
 #### Add the flags
 
+```{r}
+# Add flag column to all_data_summary_df and begin flagging data
```

Comment:
suggest integrating this flagging in the function since this won't change across site/parameter types

Reply:
Note, for efficiency and {targets}, Id' leave these two separate.

Reply:
YES I agree ... actually thinking about targets needs it's better to keep these separate.